src/data_storage/metatype_relationship_storage.ts

```diff
@@ -86,6 +86,9 @@ export default class MetatypeRelationshipStorage extends PostgresStorage{
     }
 
    public List(containerID: string, offset: number, limit:number): Promise<Result<MetatypeRelationshipT[]>> {
+        if(limit === -1) {
+            return super.rows<MetatypeT>(MetatypeRelationshipStorage.listAllStatement(containerID))
```

Comment:
This should be <MetatypeRelationshipT> right?

Reply:
What do you mean?

Reply:
Sorry this should be `<MetatypeRelationshipT>` ?